tests/suites/coslite/cl.sh

```diff
@@ -0,0 +1,71 @@
+# Ensure that COS Lite deploys successfully, the
+# relations are setup as expected and we can access the dashboards.
+run_deploy_coslite() {
+	echo
+
+	local model_name file overlay_path admin_passwd alertmanager_ip grafana_ip prometheus_ip
+	model_name="deploy-coslite"
+	file="${TEST_DIR}/${model_name}.log"
+	ensure "${model_name}" "${file}"
+
+	overlay_path="./tests/suites/coslite/overlay"
+	juju deploy cos-lite --trust --channel=edge --overlay "${overlay_path}/offers-overlay.yaml" --overlay "${overlay_path}/storage-small-overlay.yaml"
+
+	echo "Wait for all coslite bundle charms to be active"
+	wait_for "alertmanager" "$(active_condition "alertmanager" 0)" 1800
+	wait_for "grafana" "$(active_condition "grafana" 1)" 1800
+	wait_for "loki" "$(active_condition "loki" 2)" 1800
+	wait_for "prometheus" "$(active_condition "prometheus" 3)" 1800
+	wait_for "traefik" "$(active_condition "traefik" 4)" 1800
+
+	echo "Check that all relations specified in the bundle exist"
+	wait_for 5 '.applications["alertmanager"] | ."relations" | length'
+	wait_for 3 '.applications["grafana"] | ."relations" | length'
+	wait_for 5 '.applications["loki"] | ."relations" | length'
+	wait_for 6 '.applications["prometheus"] | ."relations" | length'
+	wait_for 2 '.applications["traefik"] | ."relations" | length'
+
+	admin_passwd=$(juju run-action --wait grafana/0 get-admin-password --format json | jq '.["unit-grafana-0"]["results"]["admin-password"]')
+	if [ -z "$admin_passwd" ]; then
+		echo "expected to get admin password for grafana/0"
+		exit 1
+	fi
+
+	# Assert the web dashboards are reachable
+	alertmanager_ip=$(juju status --format json | jq '.applications["alertmanager"]["units"]["alertmanager/0"].address' | tr -d '"')
+	# expect to see the main page (no redirections)
+	curl -sSL -D - http://"$alertmanager_ip":9093 -o /dev/null
+	grafana_ip=$(juju status --format json | jq '.applications["grafana"]["units"]["grafana/0"].address' | tr -d '"')
+	# expect to be redirected to the login page
+	curl -sSL -D - http://"$grafana_ip":3000 -o /dev/null
```

Comment:
I don't know if grafana has a health check. Worth doing a few mins of research

Reply:
Yes it has. Thanks once again for this.